internals/secrethub/formatter.go

```diff
@@ -153,40 +153,41 @@ func (f *tableFormatter) columnWidths() []int {
 	if f.computedColumnWidths != nil {
 		return f.computedColumnWidths
 	}
-	res := make([]int, len(f.columns))
+	fixedWidths := make([]int, len(f.columns))
```

Comment:
:+1: for renaming it to a more verbose name. Specifically in this case however, `fixed` can have a double meaning: "adjusted"/"repaired" or "not changing" (a fixed value). Changing it to something like `adjustedWidths` does away with this possible confusion.

Besides that the new comments really improve the legibility of the code :ok_hand: :